common/lib/dependabot/utils.rb

```diff
@@ -44,5 +44,15 @@ def self.always_clone_for_package_manager?(package_manager)
     def self.register_always_clone(package_manager)
       @cloning_package_managers << package_manager
     end
+
+    def self.wildcard_match?(wildcard_string, candidate_string)
```

Comment:
Question: what else is this used by/do we anticipate using this?

Could we reduce its visibility? Maybe it only needs to exist in `IgnoreCondition`

Reply:
Good shout, folded into UpdateConfig for now, thought it might be easier to refactor into IgnoreCondition once we've merged it into your branch as I don't want to create too many merge conflicts